### ISSUE TITLE
cask-repair doesn't generate SHA256 string correctly

### ISSUE THREAD

**Issue by quanglam2807** [vitorgalvao/tiny-scripts#196]
I'm using `cask-repair` with GitHub Actions and it has been working for months. But recently, `cask-repair` somehow generates an empty SHA256 string and fails the test which keeps the CI running for hours before timing out.

```
• audit for webcatalog: failed
 - sha256 string must be of 64 hexadecimal characters
• audit for webcatalog: failed
 - sha256 string must be of 64 hexadecimal characters
• audit for webcatalog: failed
 - sha256 string must be of 64 hexadecimal characters
• audit for webcatalog: failed
 - sha256 string must be of 64 hexadecimal characters
• audit for webcatalog: failed
 - sha256 string must be of 64 hexadecimal characters
• audit for webcatalog: failed
 - sha256 string must be of 64 hexadecimal characters
... (forever)
```

Here is the log:
https://github.com/atomery/webcatalog/runs/1069957677

Here is the GitHub Action workflow configuration:
https://github.com/atomery/webcatalog/blob/master/.github/workflows/cask.yml

**Comment by vitorgalvao** [comment]
From your log:

> You have set 'HOMEBREW_NO_AUTO_UPDATE'. If cask-repair fails, unset it and retry your command before submitting a bug report.

Try it locally. Also, I’ll need you to do https://github.com/vitorgalvao/tiny-scripts/issues/195#issuecomment-682061436.

This script is used by a lot of people. Unless I get more reports I’ll have to assume it’s on your end. I’ll review a PR, but I’m not taking issues of isolated cases.

**Comment by quanglam2807** [comment]
`HOMEBREW_NO_AUTO_UPDATE=1` (default GitHub Actions behavior) but I did call `brew update` before `brew install`. 

https://github.com/atomery/webcatalog/runs/1074460075#step:5:572 : I suspect the problem is here: 
```
shasum: ==> Success! Downloaded to -> /Users/runner/Library/Caches/Homebrew/downloads/bd866e91d6eb9654d643a5e607bc76a45d7434c7ea3b53002f15338965ce2d2f--WebCatalog-24.3.0-mac.zip: No such file or directory
```

I have used the same script on GitHub Actions for months and it just started to fail recently. It seems like it only affects GitHub Actions, not local environment.
